src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -59,6 +68,7 @@ def test_pf_version(self, capfd):
         out, err = capfd.readouterr()
         assert out == "0.0.1\n"
 
+    @pytest.mark.skipif(pf_recording_mode() == "replay", reason="Instable in replay mode.")
```

Comment:
how about make `pf_recording_mode() == "replay"` a function like `is_live`? so that we will have cleaner look